client/ayon_core/hosts/maya/plugins/publish/extract_pointcache.py

```diff
@@ -152,24 +202,147 @@ def process(self, instance):
     def get_members_and_roots(self, instance):
         return instance[:], instance.data.get("setMembers")
 
+    @classmethod
+    def get_attribute_defs(cls):
+        override_defs = {
+            "attr": {
+                "def": TextDef,
+                "kwargs": {
+                    "label": "Custom Attributes",
+                    "placeholder": "attr1; attr2; ...",
+                }
+            },
```

Comment:
Is there a reason why this is structured like this - why are we not just creating the `TextDef` itself but only returning it from `get_attribute_defs` if it should be exposed? Seems odd that we're going for this data intermediate structure in between? Seems more complex than needed?